ui/src/views/Hooks/ListHooks/index.jsx

```diff
@@ -46,25 +48,80 @@ export default class ListHooks extends Component {
     });
   };
 
+  updateExpanded() {
+    this.query = qs.parse(this.props.location.search.slice(1));
```

Comment:
I don't think this needs to be a method on the class. You can just extract the query string dynamically like so ` const { somethingMeaningful } = qs.parse(this.props.location.search)` and then you don't need line 56 (just use the variable you destructured in place of hookSearch).

Reply:
done